smf/clickhouse/method_script.sh

```diff
@@ -28,13 +28,135 @@ ipadm show-addr "$DATALINK/ll" || ipadm create-addr -t -T addrconf "$DATALINK/ll
 ipadm show-addr "$DATALINK/omicron6"  || ipadm create-addr -t -T static -a "$LISTEN_ADDR" "$DATALINK/omicron6"
 route get -inet6 default -inet6 "$GATEWAY" || route add -inet6 default -inet6 "$GATEWAY"
 
-args=(
+# TEMPORARY: Racks will be set up with single node ClickHouse until
+# Nexus provisions services so there is no divergence between racks
+# https://github.com/oxidecomputer/omicron/issues/732
+single_node=true
+
+command=()
+# TODO: Remove single node mode once all racks are running in replicated mode
```

Comment:
Nitpick, for here and elsewhere in this PR -- if you do:

```
$ git diff main | grep "TODO:"
```

Could we embed an appropriate issue number for these TODOs? IMO `TODO(#123)` gives a lot of context, by pointing to an issue number